tools/data-api/internal/endpoints/v1/routing.go

```diff
@@ -21,35 +21,44 @@ type Options struct {
 	UsesCommonTypes bool
 }
 
-func Router(router chi.Router, options Options) {
+type Api struct {
+	ServicesRepository repositories.ServicesRepository
+	Services           *[]string
```

Comment:
Two things:

1. These can become private fields rather than public?
2. Since this list is being parsed down to the repository, why do we need `Services` stored here too?